R/resolver.R

```diff
@@ -556,7 +556,22 @@ resolve_groups <- function(expr, vector) {
     return(":GRAND_SUMMARY:")
   }
 
+  # Handle groups = FALSE supplied to not do any summary rows.
+  if (isFALSE(resolved)) {
+    resolved <- NULL
+  }
+
   if (length(resolved) < 1) {
+    # 1292
+    input <- tryCatch(rlang::as_label(quo), error = NULL)
+    if (identical(input, "everything()")) {
```

Comment:
The only way I found to figure out if the call was from `grand_summary_rows()` or not.